### DISCUSSION TITLE
E73-2G4M04S1B nRF52832

### DISCUSSION THREAD

**Discussion by azgael** [rbaron/b-parasite#129] [Q&A]
Hello 
I have ordered by mistake the E73-2G4M04S1B (is not clear in ebay!) including a nRF52832 instead of the E73-2G4M08S1B nRF52833, basically it has half of the ram from nRF52833. 
is it possible to recompile the code for this device instead of the nRF52833?
if so can you guide me on the changes required on the board description file?
thank you very much!!
BR
Azgael

**Comment by rbaron** [comment]
Hi @azgael, unfortunately it seems like the [E73-2G4M04S1B](https://www.ebyte.com/en/product-view-news.aspx?id=243) module has a different footprint than both the [E73-2G4M08S1C](https://www.ebyte.com/en/downpdf.aspx?id=445) and [E73-2G4M08S1E](https://www.ebyte.com/en/product-view-news.html?id=1035). In order to make it work, you would have to redesign the PCB in addition to adding software support for it.
